pkg/question/question.go

```diff
@@ -1046,4 +1046,21 @@ func AskInstanceIds(h *ec2helper.EC2Helper, addedInstanceIds []string) (*string,
 	})
 
 	return &answer, err
+}
+
+// Confirm if the user wants to terminate the selected instanceIds
+func AskTerminationConfirmation(instanceIds []string) string {
```

Comment:
np: just a Go stylistic thing (usually caught by go-report-card) could you start the comment with `// AskTerminationConfirmation confirms if the user...`